packages/common/src/groups/_internal/GroupBusinessRules.ts

```diff
@@ -80,6 +81,23 @@ export const GroupPermissionPolicies: IPermissionPolicy[] = [
     authenticated: true,
     entityOwner: true,
   },
+  {
+    permission: "hub:group:canEditAccess",
+    dependencies: ["hub:group"],
+    authenticated: true,
+    assertions: [
+      {
+        property: "context:currentUser.privileges",
+        type: "contains-some",
```

Comment:
Could we use "contains" here, since we only have one thing we're checking for ("portal:admin:updateGroups")?